staging/src/k8s.io/mount-utils/mount_helper_common.go

```diff
@@ -101,6 +105,10 @@ func doCleanupMountPoint(mountPath string, mounter Interface, extensiveMountPoin
 		return err
 	}
 
+	if mounter.canSafelySkipMountPointCheck() {
```

Comment:
actually in line 103, after your change, how can we make sure it is a mountpoint? as long as mounter.canSafelySkipMountPointCheck() is true, if mountPath is not a mount point, it will return err at line 105, right?